---
permalink: /publications/
title: "Publications"
layout: single
classes: wide
author_profile: true
---


## Genomic data representations for horizontal gene transfer detection


[![HGT representations image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/hgt_representations.webp)](https://doi.org/10.1093/nargab/lqaf165){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Genomic data representations for horizontal gene transfer detection."}

Wijaya, A. J., **Anžel, A.**, Richard, H., & Hattab, G. (12 2025). *Genomic data representations for horizontal gene transfer detection*. NAR Genomics and Bioinformatics, 7(4), lqaf165. [https://doi.org/10.1093/nargab/lqaf165](https://doi.org/10.1093/nargab/lqaf165){:target="_blank"}.
{: style="font-size:0.8em;"}

---


## Persona Adaptable Strategies Make Large Language Models Tractable

[![Persona image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/persona.webp)](https://doi.org/10.1145/3711542.3711581){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Persona Adaptable Strategies Make Large Language Models Tractable."}


Hattab, G., **Anžel, A.**, Dubey, A., Ezekannagha, C., Yang, Z., & İlgen, B. (2025). *Persona Adaptable Strategies Make Large Language Models Tractable*. Proceedings of the 2024 8th International Conference on Natural Language Processing and Information Retrieval, 24–31. [https://doi.org/10.1145/3711542.3711581](https://doi.org/10.1145/3711542.3711581){:target="_blank"}.
{: style="font-size:0.8em;"}


---


## Protocol for implementing the nested model for AI design and validation in compliance with AI regulations

[![Protocol_one image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/protocol_one.webp)](https://doi.org/10.1016/j.xpro.2025.103771){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Protocol for implementing the nested model for AI design and validation in compliance with AI regulations."}

Dubey, A., Yang, Z., **Anžel, A.**, & Hattab, G. (2025). *Protocol for implementing the nested model for AI design and validation in compliance with AI regulations*. STAR Protocols, 6(2), 103771. [https://doi.org/10.1016/j.xpro.2025.103771](https://doi.org/10.1016/j.xpro.2025.103771){:target="_blank"}.
{: style="font-size:0.8em;"}


---


## Current state and future prospects of Horizontal Gene Transfer detection

[![HGT_one image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/hgt_one.webp)](https://doi.org/10.1093/nargab/lqaf005){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Current state and future prospects of Horizontal Gene Transfer detection."}

Wijaya, A. J., **Anžel, A.**, Richard, H., & Hattab, G. (02 2025). *Current state and future prospects of Horizontal Gene Transfer detection*. NAR Genomics and Bioinformatics, 7(1), lqaf005. [https://doi.org/10.1093/nargab/lqaf005](https://doi.org/10.1093/nargab/lqaf005){:target="_blank"}.
{: style="font-size:0.8em;"}


---


## Interpretable Molecular Encodings and Representations for Machine Learning Tasks

[![iCAN image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/iCAN.webp)](https://doi.org/10.1016/j.csbj.2024.05.035){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Interpretable Molecular Encodings and Representations for Machine Learning Tasks."}


Weckbecker, M., **Anžel, A.**, Yang, Z., & Hattab, G. (2024). *Interpretable Molecular Encodings and Representations for Machine Learning Tasks*. Computational and Structural Biotechnology Journal, 23, 2326–2336. [https://doi.org/10.1016/j.csbj.2024.05.035](https://doi.org/10.1016/j.csbj.2024.05.035){:target="_blank"}.
{: style="font-size:0.8em;"}


---



## Kernel-based Learning for Safe Control of Discrete-Time Unknown Systems under Incomplete Observations

[![Kernel-based Learning image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/Kernel_based.webp)](https://doi.org/10.23919/CCC63176.2024.10662288){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Kernel-based Learning for Safe Control of Discrete-Time Unknown Systems under Incomplete Observations."}

Yang, Z., Dai, X., Yang, W., İlgen, B., **Anžel, A.**, & Hattab, G. (2024). *Kernel-based Learning for Safe Control of Discrete-Time Unknown Systems under Incomplete Observations*. 43rd Chinese Control Conference (CCC), Kunming, China, 2024, pp. 6627-6632. [https://doi.org/10.23919/CCC63176.2024.10662288](https://doi.org/10.23919/CCC63176.2024.10662288){:target="_blank"}.
{: style="font-size:0.8em;"}


---


## Interactive polar diagrams for model comparison

[![Polar Diagrams image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/polar_overview.webp)](https://doi.org/10.1016/j.cmpb.2023.107843){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. Interactive polar diagrams for model comparison."}

**Anžel, A.**, Heider, D., & Hattab, G. (2023). *Interactive polar diagrams for model comparison*. Computer Methods and Programs in Biomedicine, 242, 107843. [https://doi.org/10.1016/j.cmpb.2023.107843](https://doi.org/10.1016/j.cmpb.2023.107843){:target="_blank"}.
{: style="font-size:0.8em;"}


---



## A parametric approach for molecular encodings using multilevel atomic neighborhoods applied to peptide classification

[![Parametric Approach image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/parametric_results.webp)](https://doi.org/10.1093/nargab/lqac103){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. A parametric approach for molecular encodings using multilevel atomic neighborhoods applied to peptide classification."}

Hattab, G., **Anžel, A.**, Spänig, S., Neumann, N., & Heider, D. (01 2023). *A parametric approach for molecular encodings using multilevel atomic neighborhoods applied to peptide classification*. NAR Genomics and Bioinformatics, 5(1). [https://doi.org/10.1093/nargab/lqac103](https://doi.org/10.1093/nargab/lqac103){:target="_blank"}.
{: style="font-size:0.8em;"}


---


## MOVIS: A multi-omics software solution for multi-modal time-series clustering, embedding, and visualizing tasks

[![MOVIS image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/movis_overview.webp)](https://doi.org/10.1016/j.csbj.2022.02.012){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. MOVIS: A multi-omics software solution for multi-modal time-series clustering, embedding, and visualizing tasks."}


**Anžel, A.**, Heider, D., & Hattab, G. (2022). *MOVIS: A multi-omics software solution for multi-modal time-series clustering, embedding, and visualizing tasks*. Computational and Structural Biotechnology Journal, 20, 1044–1055. [https://doi.org/10.1016/j.csbj.2022.02.012](https://doi.org/10.1016/j.csbj.2022.02.012){:target="_blank"}.
{: style="font-size:0.8em;"}

---


## The Visual Story of Data Storage: From Storage Properties to User Interfaces

[![TVSDS image]({{ site.url }}{{ site.baseurl }}/assets/images/Publication_images/tvsds_Usage_name.webp)](https://doi.org/10.1016/j.csbj.2021.08.031){:target="_blank" alt="Aleksandar Anžel's (Aleksandar Anzel's) research image. The Visual Story of Data Storage: From Storage Properties to User Interfaces."}


**Anžel, A.**, Heider, D., & Hattab, G. (2021). *The Visual Story of Data Storage: From Storage Properties to User Interfaces*. Computational and Structural Biotechnology Journal, 1(1), 1. [https://doi.org/10.1016/j.csbj.2021.08.031](https://doi.org/10.1016/j.csbj.2021.08.031){:target="_blank"}.
{: style="font-size:0.8em;"}

---
